Google.Api.Generator/ConfigProtos/snippet_index.proto

```diff
@@ -0,0 +1,316 @@
+﻿// Copyright 2021 Google Inc. All Rights Reserved.
```

Comment:
Is the proto in a repo that we could include with a submodule or similar? (We should probably ask that question about the other protos in this directory.)